### DISCUSSION TITLE
classes/class_editorinterface

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#168] [User-contributed notes]
# classes/class_editorinterface

Inherits: Object Godot editor's interface. Description: EditorInterface gives you control over Godot editor's window. It allows customizing the window, saving and (re-)loading scenes, rendering mes...

https://docs.godotengine.org/en/stable/classes/class_editorinterface.html

**Comment by donn-xx** [comment]
In gdscript, to avoid errors when exporting your game, like: `SCRIPT ERROR: Parse Error: Identifier "EditorInterface" not declared in the current scope.` Try this workaround:

```gdscript
var editor_interface = Engine.get_singleton("EditorInterface") # Avoids explicitly using the Singleton
#if not EditorInterface.is_plugin_enabled("dabber"): # This gives the error
if not editor_interface.is_plugin_enabled("dabber"): # This works okay, but you lose auto-complete
  pass
```

Suggested on [The Godot forum by "mrcdk"](https://forum.godotengine.org/t/how-to-strip-editor-specific-code-from-build-in-godot-4-2/41978/4)
